CMakeLists.txt

```diff
@@ -219,6 +219,9 @@ if (FMT_INSTALL)
   set(FMT_LIB_DIR ${CMAKE_INSTALL_LIBDIR} CACHE STRING
       "Installation directory for libraries, relative to ${CMAKE_INSTALL_PREFIX}.")
 
+  set(FMT_BIN_DIR ${CMAKE_INSTALL_BINDIR} CACHE STRING
+      "Installation directory for libraries, relative to ${CMAKE_INSTALL_PREFIX}.")
```

Comment:
Does this need to be configurable? I suggest removing this option and just using `${CMAKE_INSTALL_BINDIR}`.

Reply:
No, this does not need to be configurable. I just copied what was there for `${FMT_LIB_DIR}`.